kafka/tools/assigner/actions/balancemodules/even.py

```diff
@@ -77,10 +77,6 @@ def process_cluster(self):
                     partition = topic.partitions[pnum]
 
                     for pos in range(len(partition.replicas)):
-                        # Current placement is fine (or low). Leave the replica where it is
-                        if pmap[pos][partition.replicas[pos].id] <= target:
```

Comment:
While removing this line may not break any tests, it does represent a significant increase in processing for this balance module, as we no longer skip partitions that are already in a decent place. I'd like to understand a little more about the particular case that this is failing to work in, and why removing this line is the right fix (as opposed to altering the line to account for the case you're seeing).

Ultimately, if you have a bunch of brokers that have a count of 1, and some that have 0 and 2, the broker that has 2 should have a partition moved off of it. If the selection of what gets moved and where is not correct, we should address why that happens more directly.